### PR TITLE
A lot of minor code fixes

### PR DESCRIPTION
**Pull request by Kozova1**
A lot of code readability and correctness added

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -2,138 +2,92 @@
 
 ok="Ok, going to the next question."
 
-cat << "EOF"
+read -r -d '' dna_ascii << "EOF"
```
Comment (Kozova1) [inline init.sh:6]: Saving the DNA logo thingie into a variable (dna_ascii) to minimise duplication

File: init.sh [hunk 2]
```diff
@@ -2,138 +2,92 @@
...
 echo ""
-echo "We will ask some questions and configure according to your answers. Please reply by typping a number (according to your choice) and press [ENTER]."
-cat << "EOF"
+echo "We will ask some questions and configure according to your answers. Please reply by typing y/n (according to your choice) and press [ENTER]."
 
--. .-.   .-. .-.   .-. .-.   .
-  \   \ /   \   \ /   \   \ /
- / \   \   / \   \   / \   \
-~   `-~ `-`   `-~ `-`   `-~ `-
+echo "${dna_ascii}"
```
Comment (Kozova1) [inline init.sh:20]: use "${var}" istead of "$var" for code readability and clarity

File: init.sh [hunk 3]
```diff
@@ -2,138 +2,92 @@
...
-#    echo "$gitormanual isn't a valid answer (as we've asked 1 or 2). Restarting..."
-#    sudo ./init.sh && exit
-# fi
-cat << "EOF"
 
--. .-.   .-. .-.   .-. .-.   .
-  \   \ /   \   \ /   \   \ /
- / \   \   / \   \   / \   \
-~   `-~ `-`   `-~ `-`   `-~ `-
+case gitormanual in
```
Comment (Kozova1) [inline init.sh:30]: we can use a case statement here instead of if statements

File: init.sh [hunk 4]
```diff
@@ -2,138 +2,92 @@
...
+   [nN])
+      echo "Removing .git folder..."
+      sudo rm -r .git
+      echo "Removing update module..."
+      sudo rm modules/update
+      echo "$ok" ;;
+   
+   *)
```
Comment (Kozova1) [inline init.sh:42]: BUG: if you type anything except n or N it assumes you wrote y. Will fix soon

File: floflis [hunk 5]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # load definitions & settings
-source /usr/sbin/layers/dna/config
```
Comment (Kozova1) [inline floflis:4]: the FHS says that it is bad to put folders in /bin /usr/bin /sbin and /usr/sbin

**Review by Kozova1 — COMMENTED** [review]
a

File: init.sh (hunk 1 above)
Comment (DaniellMesquita) [reply init.sh]: Getting rid of brackets and commas/collons is why I prefer shell.
But, if this adds compatibility with other shells than BASH, ok.

File: init.sh (hunk 2 above)
Comment (danimesq) [reply init.sh:20]: Getting rid of brackets and commas/collons is why I prefer shell.
But, if this adds compatibility with other shells than BASH, ok.

File: init.sh (hunk 3 above)
Comment (danimesq) [reply init.sh:30]: Very cleaner than the previous. Thanks.

File: init.sh (hunk 4 above)
Comment (danimesq) [reply init.sh:42]: Issue: https://github.com/Floflis/Floflis-DNA/issues/2

File: floflis (hunk 5 above)
Comment (danimesq) [reply floflis:4]: I think /usr/bin is better than /usr/sbin

**Comment by danimesq** [comment]
Thanks for your PR, appreciated. It will be a code standard model for any layer of Floflis.

**Pull request merged**